tests/search/global-phase-ranking/global-phase-ranking.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+require 'indexed_search_test'
+
+class GlobalPhaseRanking < IndexedSearchTest
+
+  DOCS = 5
+
+  def setup
+    set_owner("bjorncs")
+    set_description("Test 'global-phase' ranking")
+    deploy_app(SearchApp.new.sd(selfdir + "test.sd"),
+               :files => {selfdir + "files/multiply_add.onnx" => "files/multiply_add.onnx",
+                          selfdir + "files/matrix.json" => "files/matrix.json"})
+    start
+    feed_and_wait_for_docs("test", 5, :file => selfdir + "docs.json")
+  end
+
+  def test_global_phase
+    puts "Search with second-phase ranking"
+    result_sp = search("/search/?input.query(query_vec)=[2.0,2.0]&query=sddocname:test&format=xml&ranking=second_phase")
```

Comment:
@bjorncs Any reason why _&format=xml_ is used (as that is deprecated)? Consider using default JSON format instead. Also the _/search/_ prefix is not required.

Reply:
Not really, fixed in https://github.com/vespa-engine/system-test/pull/2763.